.gitignore

```diff
@@ -27,3 +27,6 @@ dist/
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules/
 bower_components/
+
+# .DS_Store
+client/sender-app/content/.DS_Store
```

Comment:
If you do **/*.DS_Store, it should ignore DS_Store files, wherever they are in the directory structure